src/translations/nl.ts

```diff
@@ -1,4 +1,9 @@
 export default {
+    'callout.note': 'Opmerking',
+    'callout.important': 'Belangrijk',
+    'callout.tip': 'Tip',
+    'callout.example': 'Waarschuwing',
```

Comment:
This key should have been 'callout.caution'

Reply:
Ah crap… 😅

Good find! I'll fix 👍

Reply:
Fix here: https://github.com/Lundalogik/lime-elements/pull/2204